packages/pymedphys_analysis/src/pymedphys_analysis/tpscompare/bulkcompare.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2019 Cancer Care Associates
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affrero General Public License. These aditional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+import re
+import pathlib
+
+from .mephysto import absolute_scans_from_mephysto
+
+
+def load_and_normalise_mephysto(directory, regex, absolute_doses,
```

Comment:
It seems to me that this belongs in `mephysto.py`...

All functions are mephysto-specific and yet the filename "bulkcompare" implies generality.

Reply:
True, but for now I would rather leave it where it is until it is actually used by something else.

...or in the near future the code base needs a refactor anyway.